torch/csrc/jit/codegen/cuda/kernel_ir_dispatch.cpp

```diff
@@ -478,6 +478,42 @@ void OptOutDispatch::handle(GridBroadcast* stmt) {
 void OptOutDispatch::handle(GridWelford* stmt) {
   unhandled(stmt);
 }
+
+std::vector<Expr*> KirVisitor::handle(const std::vector<Expr*>& exprs) {
+  exprs_ = std::vector<Expr*>(exprs);
+  for (auto expr : exprs_) {
```

Comment:
Isn't `exprs_` potentially modified during a traversal? Shouldn't `exprs` be used here instead?